### PR TITLE
Changes behaviour of timeout

### PR DESCRIPTION
**Pull request by gummiboll**
Hey (no phun intended)

This PR changes the behaviour of timeouts. Instead of checking for timeout on TLSHandshake/dial this sets the timeout for http.Client which includes both. Changed the default value from 0 to 20 and went from ms to s which seems easier to handle for the human using hey.

Not sure if this is the way you want to go, if not you can just ignore this PR. :)

My editor also fixed some trailing whitespaces

### CODE REVIEW

File: hey.go [hunk 1]
```diff
@@ -86,7 +86,7 @@ Options:
   -m  HTTP method, one of GET, POST, PUT, DELETE, HEAD, OPTIONS.
   -H  Custom HTTP header. You can specify as many as needed by repeating the flag.
       for example, -H "Accept: text/html" -H "Content-Type: application/xml" .
-  -t  Timeout in ms.
+  -t  Timeout in seconds. Default is 20, use 0 to disable.
```
Comment (rakyll) [inline hey.go]: 0 to disable or use the default? http.Client has a default timeout.

**Comment by rakyll** [comment]
I think renaming the binary is a good excuse for such a breaking change :)

**Comment by rakyll** [comment]
Btw, is this your first time contributing to a Google project? If so, could you please review and sign the Google CLA? https://cla.developers.google.com/ to sign.

File: hey.go (hunk 1 above)
Comment (gummiboll) [inline hey.go]: It has? I was under the impression that it didn't?

**Comment by gummiboll** [comment]
Signed!

File: hey.go (hunk 1 above)
Comment (rakyll) [inline hey.go]: Oh, it says "A Timeout of zero means no timeout.". We are good.

**Comment by rakyll** [comment]
Thanks!

**Comment by gummiboll** [comment]
Thank *you* (and all contributors) for a excellent piece of software. :)

**Pull request merged**